content/en/docs/integration-guide.md

```diff
@@ -94,11 +94,23 @@ Many browsers will fetch OCSP from Let's Encrypt when they load your site. This
 
 By turning on OCSP Stapling, you can improve the performance of your website, provide better privacy protections for your users, and help Let's Encrypt efficiently serve as many people as possible.
 
-# Let's Encrypt IPs
+# Firewall Configuration
 
-Let's Encrypt will validate from a number of different IP addresses in the future, and will not announce which ones in advance. You should make sure your validation server is available to all IPs.
+To use Let's Encrypt, you need to allow outbound port 443 traffic from the
+machines running your ACME client. We don't publish the IP ranges for our
+ACME service, and they will change without notice.
 
-Some people who are issuing for non-HTTP services want to avoid exposing port 80 to anyone except Let's Encrypt's validation server. If you're in that category you may want to use the DNS challenge instead.
+For the "http-01" ACME challenge, you need to allow inbound port 80 traffic.
+We don't publish the IP ranges from which we perform validation, and they
+will change without notice.
+
+Note: We recommend always allowing plain HTTP access to your
+web server, with a redirect to HTTPS. This provides a better user
+experience than a web server that refuses or drops port 80 connections,
+and provides the same level of security.
+
+For the "dns-01" ACME challenge, you need to allow inbound port 53 traffic
+(TCP and UDP) to your DNS server.
```

Comment:
nit: "DNS servers" not "DNS server". IANA requires there be [two authoritative namservers](https://www.iana.org/help/nameserver-requirements) and I think a plurality of servers is the most common situation.

It might also be best to explicitly use the word "authoritative" to further disambiguate.

Reply:
(Removed an earlier IANA reference, its not applicable to this situation).